internal/store/leaf/templates.go

```diff
@@ -116,6 +116,9 @@ func (s *Store) SetTemplate(ctx context.Context, name string, content []byte) er
 	p := s.templatefile(name)
 
 	if err := s.storage.Set(ctx, p, content); err != nil {
+		if errors.Is(err, store.ErrMeaninglessWrite) {
+			return fmt.Errorf("no need to overwrite template")
```

Comment:
If we don't want to overwrite because it already has the right content shouldn't we consider to return nil instead?

Reply:
done